nva-datamodel-java/src/main/java/no/unit/nva/model/ImportDetail.java

```diff
@@ -2,6 +2,16 @@
 
 import com.fasterxml.jackson.annotation.JsonTypeInfo;
 import java.time.Instant;
+import no.unit.nva.model.ImportSource.Source;
 
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
-public record ImportDetail(Instant importDate, ImportSource source) { }
+public record ImportDetail(Instant importDate, ImportSource importSource) {
+
+    public static ImportDetail fromSource(Source source, Instant importDate) {
+        return new ImportDetail(importDate, ImportSource.fromSource(source));
+    }
+
+    public static ImportDetail fromBrageArchive(String archive, Instant importDate) {
+        return new ImportDetail(importDate, ImportSource.fromBrageArchive(archive));
+    }
```

Comment:
Convenient with these two methods?

So we just run 
`ImportDetail.fromBrageArchive(archive, date)` 
instead of 
`new ImportDetail(date, new ImportSource(source, archive)`